NamelessSpigot/resources/config.yml

```diff
@@ -56,6 +56,9 @@ auto-whitelist-registered:
   poll-interval: 300
   # Send log messages when whitelisting or unwhitelisting users
   log: true
+  # Keep the following players permanently whitelisted despite not having a registered or activated account
+  permanent-whitelisted:
```

Comment:
I think this should be renamed to "exclude", because the option does not keep players permanently whitelisted (if players are removed from the whitelist your code won't add them again)

Also it should use UUIDs instead of player names